README.md

```diff
@@ -9,6 +9,15 @@ dataset.
 
 xFELTOR is inspired by [xBOUT](https://github.com/boutproject/xBOUT) and uses currently some of its plotting functionality.
 
+## Why xarray?
+
+xarray is a powerful open-source python library which aims to provide Pandas-like labelling, visualisation & analysis functionality for N-dimensional data. Some of the basic features are:
```

Comment:
evtl hier auch schon den link zu xarray einfügen [xarray]( ...)

Reply:
fixed in new commit